_pages/docs/contacts/line.md

```
@@ -0,0 +1,31 @@
+title: Линия консультации
+parent: Контакты
+permalink: /contacts/line
+---
+
+# {{ page.title }}
+---
+
+Техническая поддержка и линия консультаций по работе с расширением доступны в линии
+поддержки 1С-Коннект «Сорокин и К: Управление франчайзи для УНФ» в рабочие дни с 8.00 по
+16.00 (по московскому времени). Линия доступна всем пользователям в РФ, быстрый поиск линии
+по ссылке https://sa.1c-connect.com/join/s/hxih935nhtgsi8zx3ot58yxekc
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)